src/ape_init/_cli.py

```diff
@@ -1,11 +1,47 @@
 import shutil
 from pathlib import Path
+from typing import List
 
 import click
 
 from ape.cli import ape_cli_context
 from ape.utils import github_client
 
+try:
+    import tomllib  # type: ignore[import]
+# backwards compatibility
+except ModuleNotFoundError:
+    import tomli as tomllib
+
+
+def read_dependencies_from_toml(file_path, cli_ctx) -> List[str]:
+
+    with open("./pyproject.toml", "rb") as file:
+        try:
+            data = tomllib.load(file)
+        except FileNotFoundError:
+            cli_ctx.logger.warning(
+                f"Unable to populate contnets from pyproject.toml file. {file_path} doesn't exists."
+            )
+
+        except tomllib.TOMLDecodeError:
+            cli_ctx.logger.warning(f"Error reading {file_path} file.")
+
+    # Extract the 'tool.poetry.dependencies' section
+    dependencies = data.get("tool", {}).get("poetry", {}).get("dependencies", {})
+
+    # Check each dependency to see if it starts with 'ape-', because we know these are the ape plugins
+    ape_plugins = [dep[4:] for dep in dependencies if dep.startswith("ape-")]
+
+    return ape_plugins
+
+
+def write_ape_config_yml(dependencies: List[str], file_to_write: Path):
```

Comment:
instead of writing an ape file, the ConfigManager should read the values from the pyproject.toml and treat it the same as if the config file had existed. There is not reason to configure twice, even if one is auto-generated.